client/landing/gutenboarding/components/badge/index.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	children: React.ReactElement;
+	className?: string;
+}
+
+const Badge: React.FunctionComponent< Props > = ( { children, className, ...props } ) => (
```

Comment:
Getting type error that I'm not sure how to solve:

```

client/landing/gutenboarding/onboarding-block/design-selector/index.tsx:107:14 - error TS2322: Type '{ children: any[]; className: string; }' is not assignable to type 'Props'.
  Types of property 'children' are incompatible.
    Type 'any[]' is missing the following properties from type 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any) => Component<any, any, any>)> | null) | (new (props: any) => Component<any, any, any>)>': type, props, key

107  											<Badge className="design-selector__premium-badge">
```

Reply:
Ohh it needs to be an array 7e62a2b81d

What a messy error code to just tell that. :-D